src/Surfnet/StepupMiddleware/CommandHandlingBundle/Identity/Service/EmailVerificationMailService.php

```diff
@@ -84,7 +80,6 @@ public function __construct(
         Mailer $mailer,
         Sender $sender,
         TranslatorInterface $translator,
-        Environment $twig,
```

Comment:
You will need to update the service definitions according to this change (also goes for the other services in this commit)
The Twig environment needs to be removed from the definitions you'll find for these services..